state/migration_import.go

```diff
@@ -1429,155 +1321,6 @@ func (i *importer) makeFirewallRuleDoc(firewallRule description.FirewallRule) *f
 	}
 }
 
-// StateDocumentFactory creates documents that are useful with in the state
```

Comment:
All just dead code? Quite a bit of it...

Reply:
This all moved to `state/migration_import_tasks.go`.

I'd love to move the import tasks to `state/migrations`, but we require the private docs with in state. I'm unsure how to handle this correctly. If we do find a way, this would make the package migrations movable to `core` package.